src/main/java/com/kainos/projectdrill/database/Driver.java

```diff
@@ -6,12 +6,15 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.ArrayList;
+
+import com.kainos.projectdrill.models.Framework;
 
 
 public class Driver {
 	private final String databaseUrl = "jdbc:mysql://localhost";
 	private final String databaseUser = "root";
-	private final String databasePassword = "";
```

Comment:
@CameronMcAuley we'll need to keep the password in source '' empty string for now to work in the integration environment - please change back for your next PR. 

I'll try and add a config file if I have time.